third_party/rust/platforms/v3/BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 The Chromium Authors
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/rust/cargo_crate.gni")
+
+cargo_crate("buildrs_support") {
+  crate_name = "platforms"
+  epoch = "3"
+  crate_type = "rlib"
+  crate_root = "crate/src/lib.rs"
+  sources = [
+    "//brave/third_party/rust/platforms/v3/crate/src/error.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/lib.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/platform.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/platform/platforms.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/platform/req.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/platform/tier.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target/arch.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target/endian.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target/env.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target/os.rs",
+    "//brave/third_party/rust/platforms/v3/crate/src/target/pointerwidth.rs",
+  ]
+  inputs = [
+    "//brave/third_party/rust/platforms/v3/crate/CHANGELOG.md",
+    "//brave/third_party/rust/platforms/v3/crate/README.md",
+  ]
+
+  build_native_rust_unit_tests = false
+  edition = "2018"
+  cargo_pkg_version = "3.2.0"
+  cargo_pkg_authors = "Tony Arcieri <bascule@gmail.com>"
+  cargo_pkg_name = "platforms"
+  cargo_pkg_description =
+      "Rust platform registry with information about valid Rust platforms"
+  library_configs -= [ "//build/config/compiler:chromium_code" ]
+  library_configs += [ "//build/config/compiler:no_chromium_code" ]
+  executable_configs -= [ "//build/config/compiler:chromium_code" ]
+  executable_configs += [ "//build/config/compiler:no_chromium_code" ]
+
+  # Only for usage from third-party crates. Add the crate to
```

Comment:
did you make any manual changes to this file? I think it should have the default `std` feature here, but do those get added for build deps?

Reply:
https://github.com/brave/brave-core/pull/21115/files#diff-5e1bc4ba3bae30cc90cdf73e85e7ef7cf04b0f3ca28aee9f54ffaea1095e108fR52

Reply:
no manual changes made. shouldn't the default feature be enabled automatically anyway?

Reply:
no, in the chromium build only the features that are listed explicitly will be added. Remember the defaults come from Cargo.toml and it's not using Cargo.toml in the build